### PR TITLE
Update `README.md` with license, contribution, and build details.

### PR DESCRIPTION
**Pull request by bbewan**

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://17-bb-docs.surge.sh

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+
+
+## Copyright
+
+This repo and its contents are copyright &copy; 2017 by buddybuild,
+under the Creative Common Attribution-NonCommerical-NoDerivatives 4.0
+International license ([CC BY-NC-ND
+4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/)).
+
+You are free to share, copy, and redistribute the material in this repo
```
Comment (davidascher) [inline README.md]: This seems unnecessary.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+  documentation.
+
+* Anyone can contribute to the documentation via GitHib pull requests.
+  We hope that the buddybuild community is willing to share their
+  knowledge and experience by helping us to improve the documentation.
+
+  Note: We review all pull requests, but we cannot promise to approve all
+  submissions.
+
+* Provide a working implementation of a [GitBook](https://gitbook.com/)
```
Comment (davidascher) [inline README.md]: I'm not super motivated by this one -- if we someday decided to move away from gitbook we would still want the docs to be public.

**Comment by bbdocs** [comment]
See the website here: http://17-bb-docs.surge.sh

**Pull request merged**